apps/etl-func/README.md

```diff
@@ -16,16 +16,36 @@ cp local.settings.json.example ./.local.settings.json
 cp local.settings.json.example .local.settings.json
 ```
 
-Override with your test env variables.
+### Settings
+
+| Setting                              | Type     | Required | Description                                       |
+| ------------------------------------ | -------- | -------- | ------------------------------------------------- |
+| `FUNCTIONS_WORKER_RUNTIME`           | `string` | `true`   | Must be "node". No need to be changed             |
+| `MESSAGE_CONTENT_STORAGE_URI`        | `string` | `true`   | URI of the azure storage resource                 |
+| `MESSAGE_CONTENT_CONTAINER_NAME`     | `string` | `true`   | Name of the storage container                     |
+| `MESSAGE_EVENTHUB_CONNECTION_STRING` | `string` | `true`   | Eventhub connection string from the azure portal. |
+| `MESSAGE_EVENTHUB_NAME`              | `string` | `true`   | Name of the Eventhub azure resource               |
```

Comment:
I think it is better to remove this section, let's just use the `.example` file to store all the variables

Reply:
@lucamancosu